mypy/test/teststubgen.py

```diff
@@ -804,6 +809,52 @@ def __init__(self, arg0: str) -> None:
             'def __init__(*args, **kwargs) -> Any: ...'])
         assert_equal(set(imports), {'from typing import overload'})
 
+    def test_cython(self) -> None:
+        pyx_source = """
```

Comment:
Style nit: What about indenting the source and using `textwrap.dedent()` to dedent it?